pkg/cmd/stack/translate.go

```diff
@@ -83,16 +85,39 @@ func translateStackEnvVars(s *model.Stack) error {
 	return nil
 }
 
-func translateServiceEnvFile(svc *model.Service, filename string) error {
+func translateServiceEnvFile(ctx context.Context, svc *model.Service, svcName, filename string, isOktetoNamespace bool) error {
 	var err error
 	filename, err = model.ExpandEnv(filename)
```

Comment:
Right now we are only failing when the file does not exist and we couldn't get any secrets. It could be that the user is not on a Okteto cluster or because we could not get any secret because a failure on the API.